src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 I don't know if you noticed, but there's a lot of warnings in the hardware accelerated output. It seems like most of the parameters isn't used by the encoder, and at least one thing that should be specified isn't. From what I understand the hardware accelerated encoders have very few options, as they are limited to whatever "profiles" the GPU supports. It seems like quite an undertaking to try to use hardware acceleration the way it's implemented, you'd need to know a lot about that particular GPU to know which options are valid.